userguide/image_similarity/README.md

```diff
@@ -111,6 +111,68 @@ similar_images = similarity_graph.edges
 [91440 rows x 4 columns]
 ```
 
+##### Deploying to Core ML
+
+With the Core ML framework, you can use the model to
+retrieve the distances between a query image and all images in the reference data. Exporting this model in Core ML format can be
+performed using the `export_coreml` function.
+
+```python
+model.export_coreml('MyImageSimilarityModel.mlmodel')
+```
+
+When you open the model in Xcode, it looks like the following:
+
+![Image similarity model in Xcode](images/image_similarity_model.png)
+
+The following Swift code is needed to consume the model in
+an iOS app.
+
+```swift
+lazy var classificationRequest: VNCoreMLRequest = {
+	do {
+		let model = try VNCoreMLModel(for: MyImageSimilarityModel().model)
+		
+		let request = VNCoreMLRequest(model: model, completionHandler: { [weak self] request, error in
+		    self?.processQuery(for: request, error: error)
+		})
+		request.imageCropAndScaleOption = .centerCrop
+		return request
+	} catch {
+            fatalError("Failed to load Vision ML model: \(error)")
```

Comment:
This is error message that is prepended may not be the best. There are other thing that can fail in the code block besides loading the model, right?

Reply:
Removed and made consistent with all other image toolkits.